### ISSUE TITLE
Dealing with mask output from prediction

### ISSUE THREAD

**Issue by Tengoles** [haotian-liu/yolact_edge#118]
**Describe the bug**
Not really a bug but I'm having a hard time getting the mask from the prediction in a format that I can understand.

**To Reproduce**
```
frame = cv2.imread("some_img.jpg")
frame = torch.from_numpy(frame).cuda().float()
batch = FastBaseTransform()(frame.unsqueeze(0))

with torch.no_grad():
    preds = self.net(batch, extras=self.extras)["pred_outs"]

masks = preds[0]["mask"].to("cpu").numpy().tolist()
```

**Expected behavior**
By doing the snippet shown in the code (net was previously created and is working as intended) the mask I get is in a format that I don't know how to deal with (now many elements and all of them floats between -1 and 1).

**Additional context**
As said in the beginning this is not a bug, I just need help to understand how to deal with the mask that the model outputs

**Comment by haotian-liu** [comment]
They are raw confidence scores and needs further processing for you to have them as a drawable mask.
I would like to recommend taking a look at our [Colab Notebook](https://colab.research.google.com/drive/1Mzst4q4Y-SQszIHhlEv1CkT4hwja4GNw?usp=sharing), and also the [`postprocess`](https://github.com/haotian-liu/yolact_edge/blob/master/layers/output_utils.py#L15) function which processes these values and prepare them for masks drawing onto the image.

**Comment by Tengoles** [comment]
Using the postprocess function is exactly what I was looking for, thanks!
